src/wrapper.js

```diff
@@ -40,13 +45,22 @@ export default class Wrapper extends Component {
     this.props.getStoredBookmarks();
     this.props.getRebloggedList();
     this.props.getRate();
+    this.loadMessages();
   }
 
+  loadMessages = () => {
+    steemAPI.getState('/test/@siol/4psulv-test', (err, result) => {
```

Comment:
Could be a good idea to instead of hardcoding put the string in a config file. We should do it for many other things too. It's easier to have it there in case of changing. Something like `src/config/index.js` and import as `import { translationsURL } from '../config';`